pypeit/pypmsgs.py

```diff
@@ -54,7 +54,14 @@ def __init__(self, log=None, verbosity=None, colors=True):
 
         # Initialize other variables
         self._defverb = 1
-        if getpass.getuser() in developers:
+
+        try:
+            user = getpass.getuser()
+        except ModuleNotFoundError:
+            # there appears to be a bug in getpass in windows systems where the pwd module doesn't load
+            user = os.getlogin()
+
+        if user in developers:
```

Comment:
Not really relevant to this PR, but my opinion is that we should deprecate this logging functionality.

Reply:
yes, it seemed a bit janky to me, but out-of-scope for this PR.